README.md

```diff
@@ -1,8 +1,72 @@
+I have collaborated with several labs to provide software solutions during my master's. These experiences grew interest and confidence inside
+me to contribute to the data engineering industry. Practicing programming problem-solving and taking part in programming contests is one of
+my hobbies alongside reading books on different topics.    
+
+## Skills
+
+**Proficient:** Algorithms & Data structures, Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git, Springboot, Maven
+
+**Competent:** Python, C, C++, Image Processing algorithms e.g. BM3D, Image processing tools e.g. Matlab, OpenCV
+
+**Familiar:** JavaScript, Node.js, Express.js, HTML, CSS
+
+## Development Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_zaied](https://github.com/jarviscanada/jarvis_data_eng_zaied)
+
+- **[Cluster Monitor](./linux_sql)**: Developed a `Bash` and `PostgreSQL` based cluster monitoring agent for distributed systems. The Node machines send continuous hardware usage and hardware specification data which is monitored and persisted by server machine in a PostgreSQL database. The PostgreSQL database instance was hosted by a `docker` container. The container was created based on the PostgreSQL base image from the `docker registry`. `Maven` was
+been used to manage dependencies.
+- **[Core Java Apps](./core_java)**: Developed three `Maven` managed pure `Java`-based apps. 
+    * The Twitter app shows, posts, deletes tweet data using the Twitter API. Efficient exception handling, logging, and testing frameworks were used in these apps. `self4j` facade and as the implementation `log4j` was used for the logging purpose whereas `Mockito` framework was used for unit testing and `Junit 4` was used for Integration testing. `MVC` architecture was used where `Dao` pattern was used for data access and retrieval. `Jackson` API was used to parse and create `JSON` objects to `POJO` objects or vice versa.
+    * The JDBC app interacts with the `PostgreSQL` database with pure `Java` and JDBC driver. The `PostgreSQL` database instance is hosted by a docker container that was created from the `PostgreSQL` base image.
+    * The grep app recursively traverses directories and subdirectories
+and perform searching or text-processing
+- **[SpringBoot App](./springboot)**: Developed this `maven` packaged `springboot` managed `Java` app. This is an `MVC` architecture driven `three-tier` `Microservice` REST API web app which can be consumed by any REST API client or browser. It responses back JSON object. It interacts with market data provider API, extracts stock Market data and shows, updates, deletes and additionally persists them in a dockerized `PostgreSQL` database. This app itself is also dockerized. `Springboot annotations` were used to facilitate the `IoC` container creation, bean creation, and `Dependency Injection`.
```

Comment:
Developed a stocking trading simulation REST API with Springboot

Reply:
Designed .... using MVC arch...

Reply:
- what's this project? Stock trading simulation platform? REST API
- tech stack
- how it works?